cpp/ps0008/main (13).cpp

```diff
@@ -8,58 +7,85 @@ int Decode (string msg , int key);
 
 int main()
 {
+    // prompting user for valid ciphertext and key
+    string message;
+    int key;
     
-string message ;
-int key;
-
-cout<<"enter your message \n";
-getline(cin , message);
-
-
-cout<<"enter your key \n";
-cin >>key;
-
-cout<<"message = "<<Decode(message , key)<<"\n";
+    cout<<"enter your message \n";
+    getline(cin , message); // taking message 1/1
```

Comment:
- [x] prompted user for ciphertext (encrypted message)
- [x] stored the targeted ciphertext inside a string variable